### ISSUE TITLE
Side buttons not working. Button 4 and 5 not being detected [bug]

### ISSUE THREAD

**Issue by pranusarna769** [noah-nuebling/Mac-Mouse-Fix#1009]
## Description

I got a new MX anywhere 3S and the side buttons are not being detected by the app. 

## Steps to reproduce

1. Using Mac OS Sononma 14.5

## Context

Mac Mouse Fix Version: 2.2.4
macOS Version: Mac OS Sonoma 14.5
Mouse Model: MX Anywhere 3s

## Additional Comments

–

---

### Console Logs

–

### Crash Reports

–

**Comment by daniel-manciu-p3** [comment]
Do you have Logi Options installed? I have the same mouse and I noticed that the Input Monitoring for Logi apps is too aggressive and do not allow further processing (in order words, Mac Mouse Fix does not get the key events).

Personally, I prefer only using Mac Mouse Fix, even if some Logi Options+ features are nice.

**Comment by lwgtheone** [comment]
I prefer only using Mac Mouse Fix too.
And I have the same issue(MX anywhere 3), I am using back to MX anywhere 2s now.

**Comment by zihanpan8378** [comment]
Same problem on MC anywhere 3 on version 3.0.2.
Besides, the middle button cannot be detected too.

**Comment by dingwen07** [comment]
The MX Anywhere 3/3s lacks a dedicated left-right scrolling "wheel" (because of the magnetic main scrolling wheel cannot tilt). Logitech implemented a feature to do left-right scrolling by holding the back/forward button and then scroll the main wheel.

As a result, the mouse actually does not send a "Down" signal when the back/forward button is pressed. Instead, it sends both "Down" and "Up" signals sequentially upon releasing the button. I'm uncertain if users can redefine this behavior using Logi Options+. 

Simply put, for this type of mouse, neither the OS nor the Mac Mouse Fix software can detect when the back/forward button is pressed.
